docs/modules/Queue.md

```diff
@@ -78,40 +79,41 @@ service.
        enabled: [Queue]
        config:
           Queue:
-             'type' => 'iron',
-             'host' => 'mq-aws-us-east-1.iron.io',
-             'token' => 'your-token',
-             'project' => 'your-project-id'
+             'type': 'iron',
+             'host': 'mq-aws-us-east-1.iron.io',
```

Comment:
Oh, just one thing. This file is generated and will be overridden. 
Please update the corresponding docblock in `Queue` module so the documentation was updated.

Reply:
Thanks! Updated.